spec.bs

```diff
@@ -2859,6 +2864,175 @@ This specification defines two [=policy-controlled features=] identified by the
 Issue(WICG/turtledove#522): Move from "`*`" to "`self`".
 
 
+# Handling Direct from Seller Signals # {#handling-direct-from-seller-signals}
+
+This section specifies a manner by which signals may be provided to auctions such that the signals
+are only used within their intended auction.
+
+Scripts running on the page may peform a {{WindowOrWorkerGlobalScope/fetch()}} call using the
+`{adAuctionHeaders: true}` option, which causes the value of the [:Ad-Auction-Signals:] response
+header on that request to be used to provide signals in auctions running on the page whose
+seller matches the origin of the {{WindowOrWorkerGlobalScope/fetch()}} destination.
+
+<div algorithm="fetch capture adAuctionHeaders boolean patch">
+Modify the definition of a [=request=]: 
+
+A [=request=] has an associated boolean <dfn for=request>capture-ad-auction-headers</dfn>.
+Unless stated otherwise it is false.
+</div>
+
+<div algorithm="fetch capture adAuctionHeaders RequestInit patch">
+Modify [[FETCH]]'s {{RequestInit}} dictionary to add an adAuctionHeaders attribute:
+
+<pre class=idl>
+partial dictionary RequestInit {
+  boolean adAuctionHeaders;
+};
+</pre>
+</div>
+
+<div algorithm="fetch new request patch">
+The following step will be added to the <code><a constructor for="Request" lt="Request()">new Request (<var ignore>input</var>, <var ignore>init</var>)</a></code>
+constructor steps, before step "Set this's [=Request/request=] to |request|":
+
+1. If <var ignore=''>init</var>["{{RequestInit/adAuctionHeaders}}"] <a for=map>exists</a>, then set |request|'s
+  [=request/capture-ad-auction-headers=] to it.
+
+</div>
+
+<div algorithm="fetch Sec-Ad-Auction-Fetch patch">
+The following step will be added to the [=http-network-or-cache fetch=] algorithm, before step
+"Modify |httpRequest|'s [=request/header list=] per HTTP. ...":
+
+1. If [=request=]'s [=request/capture-ad-auction-headers=] is true, then
+  [=header list/set a structured field value=] given «[:Sec-Ad-Auction-Fetch:], the [=structured header/boolean=] `?1`» in
+  |httpRequest|'s [=header list=].
+
+</div>
+
+<div algorithm="fetch per traversable navigable structures patch">
+Modify [[FETCH]]'s [[FETCH#infrastructure]] to add a new section called "Per Traversable Navigable Structures",
+with the following content:
+
+<h3 id=direct-from-sellers-signals-key-struct>Direct from seller signals key</h3>
+A <dfn>direct from seller signals key</dfn> is a [=struct=] with the following [=struct/items=]:
+
+<dl dfn-for="direct from seller signals key">
+: <dfn>seller</dfn>
+:: An [=origin=]. Matches the origin that served the captured [:Ad-Auction-Signals:] header.

```

Comment:
I think mentioning that this is specifically related to `fetch()` responses helps clear this up a bit. I guess the part I was unclear about is, I am left to assume that these structs get created during the processing of a `fetch()` response, which I can learn by reading the https://pr-preview.s3.amazonaws.com/WICG/turtledove/771/a5f6630...caraitto:a95a01a.html#ad-auction-signals-header section, but it might be nice to link to that section directly in your Note.